planner/core/logical_plan_builder.go

```diff
@@ -5502,6 +5502,12 @@ func getInnerFromParenthesesAndUnaryPlus(expr ast.ExprNode) ast.ExprNode {
 	if uexpr, ok := expr.(*ast.UnaryOperationExpr); ok && uexpr.Op == opcode.Plus {
 		return getInnerFromParenthesesAndUnaryPlus(uexpr.V)
 	}
+	if sqpr, ok := expr.(*ast.SubqueryExpr); ok && sqpr.Query != nil {
+		if selSt, ok := sqpr.Query.(*ast.SelectStmt); ok && selSt.From == nil &&
+			len(selSt.Fields.Fields) > 0 {
+			return getInnerFromParenthesesAndUnaryPlus(selSt.Fields.Fields[0].Expr)
+		}
+	}
```

Comment:
This check would break the function name, besides, this function is not only used by `only_full_group_by` checker. I think you should add this `SubqueryExpr` check in `checkExprInGroupByOrIsSingleValue` instead.

Reply:
I think checkExprInGroupByOrIsSingleValue' job is check but not get Expr.i want to add a func named getInnerExpr to do getInnerFromParenthesesAndUnaryPlus and getSubQueryField

Reply:
I think it's ok, PTAL @eurekaka